static/app/components/profiling/ProfilingOnboarding/profilingOnboardingSidebar.tsx

```diff
@@ -83,23 +83,21 @@ export function ProfilingOnboardingSidebar(props: CommonSidebarProps) {
   }, [pageFilters.selection.projects, supportedProjects, unsupportedProjects]);
 
   const projectSelectOptions = useMemo(() => {
-    const supportedProjectItems: SelectValue<Project>[] = supportedProjects.map(
+    const supportedProjectItems: SelectValue<string>[] = supportedProjects.map(
       project => {
         return {
-          key: project.id,
-          value: project,
+          value: project.id,
           label: (
             <StyledIdBadge project={project} avatarSize={16} hideOverflow disableLink />
           ),
         };
       }
     );
 
-    const unsupportedProjectItems: SelectValue<Project>[] = unsupportedProjects.map(
+    const unsupportedProjectItems: SelectValue<string>[] = unsupportedProjects.map(
       project => {
         return {
-          key: project.id,
-          value: project,
+          value: project.id,
```

Comment:
`project` is an object. Passing an object as `value` will cause issues with selection state management. Instead, we should use `project.id`, which is a unique string.